src/app/core/models/issue.model.ts

```diff
@@ -123,7 +123,8 @@ export class Issue {
     this.title = githubIssue.title;
     this.hiddenDataInDescription = new HiddenData(githubIssue.body);
     this.description = Issue.updateDescription(this.hiddenDataInDescription.originalStringWithoutHiddenData);
-    this.milestone = githubIssue.milestone ? new Milestone(githubIssue.milestone) : null;
+    // githubIssue without milestone will be set to default milestone
+    this.milestone = githubIssue.milestone ? new Milestone(githubIssue.milestone) : Milestone.DefaultMilestone;
```

Comment:
With this change I think we are safe the assume that the milestone property will always be non-null/defined. Let's remove the optional parameter in line 30 to reflect the changes.
```milestone?: Milestone;``` -> ```milestone: Milestone```